manual/ec2-sru-20.2.45.txt

```diff
@@ -0,0 +1,2956 @@
+++ query='Reservations[].Instances[].InstanceId'
+++ run_describe_command ec2-3-21-230-147.us-east-2.compute.amazonaws.com 'Reservations[].Instances[].InstanceId'
+++ dnsname=ec2-3-21-230-147.us-east-2.compute.amazonaws.com
+++ query='Reservations[].Instances[].InstanceId'
+++ aws ec2 describe-instances --filter Name=dns-name,Values=ec2-3-21-230-147.us-east-2.compute.amazonaws.com --query 'Reservations[].Instances[].InstanceId'
+++ jq -r '.[]'
++ ec2_inst_id=i-023249c9384517e31
++ aws ec2 terminate-instances --instance-ids i-023249c9384517e31
+{
+    "TerminatingInstances": [
+        {
+            "CurrentState": {
+                "Code": 32,
+                "Name": "shutting-down"
+            },
+            "InstanceId": "i-023249c9384517e31",
+            "PreviousState": {
+                "Code": 16,
+                "Name": "running"
+            }
+        }
+    ]
+}
++ echo '### END bionic'
+### END bionic
++ for series in $SRU_SERIES
++ echo '### BEGIN eoan'
+### BEGIN eoan
+++ launch-ec2 --series eoan -u sethostname.yaml -t t2.small
+++ awk '/Found/{print $5}'
++ EC2_INST=ubuntu@ec2-3-133-105-240.us-east-2.compute.amazonaws.com
++ '[' -n ubuntu@ec2-3-133-105-240.us-east-2.compute.amazonaws.com ']'
++ sleep 60
++ trap 'terminate_instance ubuntu@ec2-3-133-105-240.us-east-2.compute.amazonaws.com' ERR
++ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR ubuntu@ec2-3-133-105-240.us-east-2.compute.amazonaws.com -- cloud-init status --wait --long
+
+status: done
+time: Thu, 11 Jun 2020 15:09:57 +0000
+detail:
+DataSourceEc2Local
++ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR ubuntu@ec2-3-133-105-240.us-east-2.compute.amazonaws.com -- cat /run/cloud-init/result.json
+{
+ "v1": {
+  "datasource": "DataSourceEc2Local",
+  "errors": []
+ }
+}
++ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR ubuntu@ec2-3-133-105-240.us-east-2.compute.amazonaws.com -- '!' grep Trace '/var/log/cloud-init.*'
++ echo 'Expect IMDSv2 token headers are being used on all metadata gets. Expect 0 line count'
+Expect IMDSv2 token headers are being used on all metadata gets. Expect 0 line count
++ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR ubuntu@ec2-3-133-105-240.us-east-2.compute.amazonaws.com -- '[[ 0 = `grep headers /var/log/cloud-init.log  | grep -c -v X-aws` ]] || exit 1'
++ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR ubuntu@ec2-3-133-105-240.us-east-2.compute.amazonaws.com -- sudo systemd-analyze
+Startup finished in 3.347s (kernel) + 1min 818ms (userspace) = 1min 4.165s 
+graphical.target reached after 1min 56ms in userspace
++ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR ubuntu@ec2-3-133-105-240.us-east-2.compute.amazonaws.com -- sudo systemd-analyze blame
+         38.382s snapd.seeded.service
```

Comment:
This is just nasty, and I believe we are aware of the snap seeding costs on eoan. but waiting 38 seconds for boot is really painful.

Reply:
Yes, I look at the past version we did an SRU and the increase seems really big:
https://github.com/cloud-init/ubuntu-sru/blob/master/manual/ec2-sru-19.4.33.txt#L1208

:cry: